test/test_jit_cuda_fuser.py

```diff
@@ -3374,27 +3374,28 @@ def __init__(self):
                 with torch.no_grad():
                     self.bias.fill_(10)
 
-            def forward(self, inputs : torch.Tensor, view_shape : List[int]):
+            def forward(self, inputs : torch.Tensor, bias : torch.Tensor, view_shape : List[int]):
                 o = inputs.view(view_shape)
-                inputs = inputs * self.bias
+                inputs.add_(bias)
```

Comment:
Just curious: any reason we want to change this to an inplace update?

Reply:
It wasn't creating an aliased operation.